pdns/lua-recursor4.cc

```diff
@@ -343,6 +343,12 @@ RecursorLua4::RecursorLua4(const std::string& fname)
   d_lw->registerFunction("getRecords", &DNSQuestion::getRecords);
   d_lw->registerFunction("setRecords", &DNSQuestion::setRecords);
 
+  d_lw->writeFunction("addPolicyTag", [](vector<string>* tags, const string& tag) {
+      if (tags) {
+        tags->push_back(tag);
+      }
+    });
+
```

Comment:
Yes. I couldn't find a way to make policytags directly writable from Lua, but I'm open to all suggestion!